### PR TITLE
add tag entity

### PR DESCRIPTION
**Pull request by Dzmitry92**
* created entity `Tag`;
* added the necessary CRUD endpoints;

### CODE REVIEW

File: packages/web-app/web-app-back/src/tag/dto/tag.dto.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import Joi from 'joi';
+
+export interface CreateTagDto {
```
Comment (SerafimovichEugene) [inline packages/web-app/web-app-back/src/tag/dto/tag.dto.ts:3]: I think just TagDto

File: packages/web-app/web-app-back/src/tag/tag.service.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    const tag = await this.getTag(id);
+    if (tag) {
+      this.tagRepository.merge(tag, bodyDto);
+      return await this.tagRepository.save(tag);
+    } else {
+      return null;
+    }
+  }
+
+  async hardDeleteTag(id: string) {
```
Comment (SerafimovichEugene) [inline packages/web-app/web-app-back/src/tag/tag.service.ts:37]: just `deleteTag`

**Review by SerafimovichEugene — DISMISSED** [review]

File: packages/web-app/web-app-back/src/tag/dto/tag.dto.ts (hunk 1 above)
Comment (Dzmitry92) [reply packages/web-app/web-app-back/src/tag/dto/tag.dto.ts]: Done.

File: packages/web-app/web-app-back/src/tag/tag.service.ts (hunk 2 above)
Comment (Dzmitry92) [reply packages/web-app/web-app-back/src/tag/tag.service.ts]: Done.

**Review by SerafimovichEugene — APPROVED** [review]

**Pull request merged**